bigbluebutton/bigbluebutton
# Added daily log rotation in production env

Added npm package `winston-daily-rotate-file`, Addresses #2939 

To test - change 
`datePattern: '.yyyy-dd-MM',`
to
`datePattern: '.yyyy-dd-MM-mm',`
This way the logs will rotate every minute (easier to test)

Also consider increasing the settings-production.json loglevel to 'verbose'

Start the HTML5 client in production mode via 

```
rm -r node_modules
meteor reset
meteor npm install
NODE_ENV=production npm start
```
Use the client for a few minutes and then look at the log files for production:
`ls -l ll /var/log/bigbluebutton/html5/` and make sure that there are several files, containing a minute-length of logs
